src/uct/ib/mlx5/ib_mlx5.inl

```diff
@@ -147,6 +147,26 @@ uct_ib_mlx5_txwq_validate(uct_ib_mlx5_txwq_t *wq, uint16_t num_bb)
 }
 
 
+static UCS_F_ALWAYS_INLINE void
+uct_ib_mlx5_txwq_update_flags(uct_ib_mlx5_txwq_t *txwq, uint32_t flags_add,
+                              uint32_t flags_remove)
+{
+#if UCS_ENABLE_ASSERT
+    txwq->flags = (txwq->flags | flags_add) & ~flags_remove;
+#endif
+}
+
+
+static UCS_F_ALWAYS_INLINE void
+uct_ib_mlx5_txwq_check_not_failed(uct_ib_mlx5_txwq_t *txwq, uint8_t opcode)
```

Comment:
1. no need this wrapper 
2. in common_post_send(): 
   if (opcode != NOP) { assert(...) }

Reply:
done